### PR TITLE
admin2: replace give admin button with permissions widget

### PR DESCRIPTION
**Pull request by N3xT**
As requested in #107

perms

### CODE REVIEW

File: [admin]/admin2/client/widgets/admin_permissions.lua [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+function aPermissions.onClick(button)
+    if (button == "left") then
+        if (source == aPermissions.Update) then
+            aPermissions.Refresh()
+        elseif (source == aPermissions.Give) then
+            if (guiGridListGetSelectedItem(aPermissions.List) == -1) then
+                messageBox("No group selected!", MB_WARNING)
+            else
+                local group = guiGridListGetItemText(aPermissions.List, guiGridListGetSelectedItem(aPermissions.List), 1)
+                if (messageBox('Are you sure to give "' .. group .. '"?', MB_QUESTION, MB_YESNO)) then
```
Comment (jlillis) [inline [admin]/admin2/client/widgets/admin_permissions.lua:77]: I think a better message would be "Are you sure you want to add [player] to the '[group name]' group?"

File: [admin]/admin2/client/widgets/admin_permissions.lua [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                    triggerServerEvent("aPlayer", getLocalPlayer(), getSelectedPlayer(), "setgroup", true, group)
+                end
+            end
+        elseif (source == aPermissions.Revoke) then
+            if (guiGridListGetSelectedItem(aPermissions.List) == -1) then
+                messageBox("No group selected!", MB_WARNING)
+            else
+                local group = guiGridListGetItemText(aPermissions.List, guiGridListGetSelectedItem(aPermissions.List), 1)
+                if (messageBox('Are you sure to revoke "' .. group .. '"?', MB_QUESTION, MB_YESNO)) then
```
Comment (jlillis) [inline [admin]/admin2/client/widgets/admin_permissions.lua:86]: I think a better message would be "Are you sure you want to remove [player] from the '[group name]' group?"

File: [admin]/admin2/conf/messages.xml [hunk 3]
```diff
@@ -93,14 +93,14 @@
       <log>ADMIN: $admin has given a jetpack to $player</log>
     </group>
     <group action="adminr" r="255" g="0" b="0">
-      <admin>$player's admin rights have been revoked</admin>
-      <player>$admin has revoked your admin rights</player>
-      <log>ADMIN: $admin has revoked admin privilegies from $player</log>
+      <admin>$player's $data group have been revoked</admin>
+      <player>$admin has revoked your $data group</player>
+      <log>ADMIN: $admin has revoked $data group from $player</log>
```
Comment (jlillis) [inline [admin]/admin2/conf/messages.xml:98]: Again, I think it would be better to say "[player] has been removed from the [group] group."

**Review by jlillis — CHANGES_REQUESTED** [review]
This PR currently contains merge conflicts. If you can resolve them and address the grammar of some of the messages then I think this is ready to merge.

**Comment by iDannz-Breno** [comment]
if someone is going to finish this, it may be worth adding the changes added in #324

**Comment by jlillis** [comment]
Going to close this PR as it has become stale. Feel free to reopen it in the future if you find time to address the above comments.

**Pull request closed without merging**